controllers/chat/findOrCreateChannelBySignedSender.js

```diff
@@ -28,13 +28,18 @@ const findOrCreateChannelBySignedSender = async (req, res) => {
 
     const findOrCreateChannel = await channel.create();
     const channelName = findOrCreateChannel.members.map((member) => member.user.name).join(', ');
-    await channel.updatePartial({
-      set: {
-        channelType,
-        channel_type: channelType,
-        name: channelName
-      }
-    });
+
+    try {
+      await channel.updatePartial({
+        set: {
+          channelType,
+          channel_type: channelType,
+          name: channelName
+        }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There seems to be a redundancy in setting `channelType` and `channel_type`. If they are meant to represent the same data, consider using only one of them to avoid confusion and potential bugs.

```diff
-        set: {
-          channelType,
-          channel_type: channelType,
-          name: channelName
-        }
+        set: {
+          channelType,
+          name: channelName
+        }
```
Please ensure that this change does not affect other parts of your codebase that may rely on the `channel_type` property.


<!-- This is an auto-generated comment by OSS CodeRabbit -->